highfidelity/hifi
# Fixing crash on appending a point to an entity

MS [#12473](https://highfidelity.fogbugz.com/f/cases/12473/Interface-crash-appending-a-point-to-a-Line-entity-at-start-up)

**TEST PLAN**
- Unzip JS file in [appendPointCrash.js.zip](https://github.com/highfidelity/hifi/files/2151023/appendPointCrash.js.zip)
- Launch Interface
- Load script from local directory in `Edit > Running Scripts...` dialog using the `FROM DISK` button
- Close Interface
- Relaunch Interface
- Interface should not crash
